app/views/finance/financial_transactions/index.html.haml

```diff
@@ -8,7 +8,18 @@
     %strong= t('.balance', balance: number_to_currency(@ordergroup.account_balance))
     %br/
     %small= t('.last_updated_at', when: distance_of_time_in_words(Time.now, @ordergroup.account_updated))
+    %p
+    - unless FoodsoftConfig[:charge_members_manually]
+      %p
+        %b= heading_helper(Ordergroup, :available_funds) + ':'
+        = number_to_currency(@ordergroup.get_available_funds())
+    - if FinancialTransactionClass.has_multiple_classes
+      - FinancialTransactionClass.sorted.each do |c|
+        %p
+          %b= c.display + ':'
+          = number_to_currency(@ordergroup["sum_of_class_#{c.id}"])
+
```

Comment:
is this the usability improvement?  it would be nice if it has some explanation on what it is improving.

Reply:
it just adds the sums for every financial transaction class